LOAN/LoanTypes/MortgageLoans.rdf

```diff
@@ -152,57 +519,234 @@
+				<owl:onClass rdf:resource="&fibo-loan-typ-mtg;PropertyInspection"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-arr-asmt;hasAppraiser"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-arr-asmt;Appraiser"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-mtg;hasAppraisedValue"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>real property appraisal</rdfs:label>
+		<skos:definition>value assessment that estimates the amount of money a real estate property is worth</skos:definition>
+		<skos:editorialNote>Ensure that the appraiser parties have NMLS Identifier and contact preferences, e.g. phone number and address. As needed, connect appraisers to an appraisal management company.</skos:editorialNote>
+		<skos:editorialNote>Future consideration: add information about variability analysis and maybe the algorithm used.</skos:editorialNote>
+		<fibo-fnd-utl-av:explanatoryNote>The valuation uses one or more methodologies and is conducted by an appraiser or technology with a logical model that performs the same function.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-loan-typ-mtg;ReverseMortgage">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;Mortgage"/>
+		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-mtg;Mortgage"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-ln-ln;hasCreditLimit"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">reverse mortgage</rdfs:label>
+		<rdfs:label>reverse mortgage</rdfs:label>
 		<skos:definition xml:lang="en">Designed for senior citizens who want to borrow against the equity of house. The house has been paid for. How is it different?: Scenario based. If you have someone who is asset rich but cash poor then a traditional view of the mortgage wqould say yoiu must have th ecash to repay the loan, which would not apply in this case: we actually don&apos;t anticipate that you would repay it, maybe not even regular amounts on the loan, but when the loan expires, e.g. when you die, we will recover on the asset in the normal course of events. Not seen as a default but is an expectation that we will recover our funds. See Wikipedia. AKA Lifetime morgtage, seniores (age xxx or older) to release the home equity.</skos:definition>
+		<skos:definition>mortgage contract that pays out money to the borrower against a set principal limit that is based on the value of existing equity in the underlying collateral.</skos:definition>
```

Comment:
need to delete the original typo-strewn definition

Reply:
done - thanks